### PR TITLE
feat(primitive): modal component

### PR DESCRIPTION
**Pull request by tarnas14**
Description
---

modal with backdrop with two sizes (small and large)

Motivation and Context
---

#93 

How Has This Been Tested?
---

modal

### CODE REVIEW

File: applications/launchpad_v2/src/components/Modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { ModalContainer, Backdrop, ModalContent } from './styles'
+import type { ModalProps } from './types'
+
+const Modal = ({ open, children, onClose, size }: ModalProps) => {
```
Comment (tomaszantas) [inline applications/launchpad_v2/src/components/Modal/index.tsx:4]: Missing JSDoc

**Review by tomaszantas — APPROVED** [review]
Looks good, the JSDoc for the Modal component is missing.

Also, I'm wondering how we want to handle `overflows` in modals. 

image

Is it better to have scrolling within the modal or the window? 

Approving this PR, the JSDoc can be added with another PR.

File: applications/launchpad_v2/src/components/Modal/index.tsx (hunk 1 above)
Comment (tarnas14) [reply applications/launchpad_v2/src/components/Modal/index.tsx:4]: will add in wallet settings

**Comment by tarnas14** [comment]
@tomaszantas didn't think of overflows, tbh, I'm not sure we have scrolling in modals, only in the settings - but then only the content will scroll (without footer and menu)

**Pull request merged**